doc/chilled.rst

```diff
@@ -0,0 +1,67 @@
+.. currentmodule:: message_ix_buildings.chilled
+
+CHILLED (:mod:`.chilled`)
+*************************
+
+:mod:`.chilled` (Cooling and Heating gLobaL Energy Demand) is a model that assesses the impacts of climate change (temperature change)
+
+Methods
+=======
+
+
+Data
+====
+
+Input data
+----------
+
+
+Parameters
+----------
+
+In this module, the "version" refers to the version of the model, not the version of the data. For example, the "ALPS2023" version refers to the set of scenario inputs/parameters for the "ALPS2023" run.
+When specifying a version, CHILLED will look in the `/data/chilled/version` directory that the version name exists and that the following files exist as well:
+- `arch_input_reg.xlsx` (or `arch_input.xlsx`)
+- `arch_regions.xlsx`
+- `par_var.csv`
+- `runs.csv`
+
+Usage
+=====
+
+The preprocessing (preparing MESSAGE region rasters and country codes) happens without specifying GCM or RCP scenario. Therefore, it can be run first on its own using the following command (from the within :mod:`chilled.run`):
```

Comment:
```suggestion
The preprocessing (preparing MESSAGE region rasters and country codes) happens without specifying GCM or RCP scenario. Therefore, it can be run first on its own using the following command (from within :mod:`chilled.run`):
```